### ISSUE TITLE
Add volume to convex hull computations

### ISSUE THREAD

**Issue by haudren** [scipy/scipy#5013]
I just noticed that no routine was available to compute the volume of a convex hull using qhull whereas these functions are available in qhull.

Unfortunately, I could not contribute directly to a pull request given that on my system Cython 0.22 is not yet available but the following seems to do more or less what I want :

```
diff --git a/scipy/spatial/qhull.pyx b/scipy/spatial/qhull.pyx
index c369ae9..e52e44a 100644
--- a/scipy/spatial/qhull.pyx
+++ b/scipy/spatial/qhull.pyx
@@ -121,6 +121,8 @@ cdef extern from "qhull/src/libqhull.h":
         realT max_outside
         realT MINoutside
         realT DISTround
+        realT totvol
+        realT totarea
         jmp_buf errexit
         setT *other_points
         unsigned int visit_id
@@ -176,6 +178,9 @@ cdef extern from "qhull/src/io.h":
 cdef extern from "qhull/src/geom.h":
     pointT *qh_facetcenter(setT *vertices) nogil
 
+cdef extern from "qhull/src/geom.h":
+    double qh_getarea(facetT *facetlist) nogil
+
 cdef extern from "qhull/src/poly.h":
     void qh_check_maxout() nogil
 
@@ -330,6 +335,18 @@ cdef class _Qhull:
             _qhull_lock.release()
 
     @cython.final
+    def volume(self):
+      global _active_qhull
+      qh_getarea(self._saved_qh.facet_list)
+      return self._saved_qh.totvol
+
+    @cython.final
+    def area(self):
+      global _active_qhull
+      qh_getarea(self._saved_qh.facet_list)
+      return self._saved_qh.totarea
+
+    @cython.final
     def close(self):
         if _qhull_lock.acquire(False):
             try:
```
But I think there is a better way to do it, as this patch segfaults the test when trying to compute an area...

If someone could guide me on the right track, I would be glad.

Thanks for your work !
Hervé Audren

**Comment by pv** [comment]
That would be an useful addition.
.
How to do it: (i) you need to lock and activate the qhull instance; see
for example get_paraboloid_shift_scale() how to do this. (ii) The area
and volume should be extracted in ConvexHull._update, as the Qhull data
structures are discared after computation except in the incremental mode.

**Comment by larsmans** [comment]
Fixed by merging of #5014.
